website/docs/Installation.md

```diff
@@ -1,10 +1,53 @@
 # Installation
 
-## Setup Virtual Environment
+## Option 1: Install and Run AutoGen in Docker
+#### Step 1. Install [docker](https://www.docker.com/).
+
+Docker is recommended as a properly built docker image could provide isolated and consistent environment to run your code securely across platforms. Install docker following [this instruction](https://docs.docker.com/get-docker/).
+
+For Mac users, alternatively you may choose to install [colima](https://smallsharpsoftwaretools.com/tutorials/use-colima-to-run-docker-containers-on-macos/) to run docker containers, if there is any issues with starting the docker daemon.
+
+#### Step 2. Build a docker image
+
+AutoGen provides [dockerfiles](https://github.com/microsoft/autogen/tree/main/samples/dockers/) that could be used to built docker images. Use the following command line to build a docker image named `autogen_img` (or some other name you like) one of the provided dockerfiles named `Dockerfile.base`:
+```
+docker build -f samples/dockers/Dockerfile.base -t autogen_img https://github.com/microsoft/autogen.git
+```
+which includes some common python libraries and essential dependencies of AutoGen, or build from `Dockerfile.full` which include additional dependencies for more advanced features of AutoGen with the following command line:
+
+```
+docker build -f sample/docker/Dockerfile.full -t autogen_full_img https://github.com/microsoft/autogen.git
```

Comment:
1. should be `samples/dockers/Dockerfile.full`.
2. I cannot run this command but the following works in autogen root directory.
`docker build -f sample/docker/Dockerfile.full -t autogen_full_img .`

Reply:
1 addressed. Thanks!
2 since the folders are added in this PR and not yet in the main repo, the link is not working right now. It will work once this PR is merged.  This should work now: `docker build -f samples/dockers/Dockerfile.base -t autogen_img https://github.com/microsoft/autogen.git#add-docker`